recipes/libssh2/all/conanfile.py

```diff
@@ -97,9 +97,6 @@ def requirements(self):
             else:
                 self.requires("mbedtls/2.28.4")
 
-    def build_requirements(self):
-        self.tool_requires("cmake/[>=3.20 <4]")
```

Comment:
As per @jcar87, this is in fact needed for 1.11 onwards for the upstream CMakelists to work with Conan-generated targets